artifactory_test.go

```diff
@@ -5807,3 +5822,29 @@ func downloadModuleAndVerify() clientutils.ExecutionHandlerFunc {
 		return false, nil
 	}
 }
+
+func sendArtifactoryTrustedPublicKey(artHttpDetails httputils.HttpClientDetails) {
+	// Send trusted public key to Artifactory
+	publicKeyPath := filepath.Join(tests.GetTestResourcesPath(), "lifecycle", "keys", "public.txt")
+	publicKey, err := os.ReadFile(publicKeyPath)
+	coreutils.ExitOnErr(err)
+	client, err := httpclient.ClientBuilder().Build()
+	coreutils.ExitOnErr(err)
+	requestBody := fmt.Sprintf(inttestutils.ArtifactoryGpgKeyCreatePattern, publicKey)
+	resp, body, err := client.SendPost(*tests.JfrogUrl+"artifactory/api/security/keys/trusted", []byte(requestBody), artHttpDetails, "")
+	if err = errorutils.CheckResponseStatusWithBody(resp, body, http.StatusCreated, http.StatusConflict); err != nil {
+		log.Error(err.Error())
+		os.Exit(1)
+	}
+}
+
+func deleteReceivedReleaseBundle(bundleName, bundleVersion string) {
+	client, err := httpclient.ClientBuilder().Build()
+	coreutils.ExitOnErr(err)
```

Comment:
Since this is the end of the test, maybe we can just `assert.NoError()` here instead of exiting (which will prevent the clean up).